init/common.js

```diff
@@ -202,6 +206,19 @@ const createProject = async (interestId, userId, project, addTeam=true) => {
   });
 };
 
+const createOpportunity = async (interestId, opportunity) => {
+    let existing = await strapi.entityService.findMany("api::opportunity.opportunity", {filters: {title: opportunity.title}});
+    existing = existing[0];
```

Comment:
I checked here and if this:
`
await strapi.entityService.findMany("api::opportunity.opportunity", {filters: {title: opportunity.title}});
` is null. When you try to access that, may it return an error:

```
> existing = null
null
> existing[0]
Uncaught TypeError: Cannot read properties of null (reading '0')
```

Reply:
@tiagopazhs good catch. Handled exception.